### ISSUE TITLE
[REQUEST] getting_started.py: I'd like to add some sort of unique (sequential) number to the `logger.info()` msgs

### ISSUE THREAD

**Issue by mwherman2000** [kdenhartog/indy-dev#17]
@kdenhartog I'd like to modify getting_started.py in a small way by adding some sort of unique (sequential) number to the `logger.info()` msgs to make it easier to correlate:
1. The code in the python script
2. The `logger.info()` msgs generated by the script
3. Written documentation about the script (e.g. https://github.com/hyperledger/indy-sdk/blob/master/doc/getting-started/getting-started.md)
4. Graphics that help visualize the steps in the script.

I want to run this by your first.  I can make the changes (and I've become quite intimate with the flow of the script).  I have a series of diagrams for about the first 60% of the python code.

Thoughts? OK to go ahead?

**Comment by kdenhartog** [comment]
Yeah, I'll definitely work with you to get this in if you add the details.

**Comment by mwherman2000** [comment]
Confirmed thru a side conversation with Kyle.
